src/nls/it/strings.js

```diff
@@ -585,21 +584,21 @@ define({
     "CMD_RESTART_NODE"                          : "Riavvia Node",
     "CMD_SHOW_ERRORS_IN_STATUS_BAR"             : "Mostra errori nella barra di stato",
     "CMD_OPEN_BRACKETS_SOURCE"                  : "Apri {APP_NAME} sorgente",
-    "ERROR_CREATING_LAUNCH_SCRIPT"              : "Si è verificato un errore durante la creazione {APP_NAME} strumento a riga di comando al <code>/usr/local/bin</code>. Per favore riferisci a <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments#troubleshooting'>linea di comando</a> wiki per la risoluzione dei problemi.<br/><br/>Motivo: ",
+    
+    "CREATING_LAUNCH_SCRIPT_TITLE"              : "Scorciatoia Riga di comando di {APP_NAME}",
+    "ERROR_CREATING_LAUNCH_SCRIPT"              : "Si è verificato un errore durante la creazione della funzione Riga di comando. Per favore riferisci a <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments#troubleshooting'>linea di comando</a> wiki per la risoluzione dei problemi.<br/><br/>Motivo: ",
     "ERROR_CLTOOLS_RMFAILED"                    : "Impossibile rimuovere collegamento simbolico esistente {APP_NAME} a <code>/usr/local/bin.</code>",
-    "ERROR_CLTOOLS_MKDIRFAILED"                 : "Impossibile creare <code>/usr/local/bin</code> struttura di directory.",
-    "ERROR_CLTOOLS_LNFAILED"                    : "Impossibile creare {APP_NAME} collegamento simbolico a <code>/usr/local/bin</code>.",
-    "ERROR_CLTOOLS_SERVFAILED"                  : "Impossibile creare l'oggetto autorizzato.",
-    "ERROR_CLTOOLS_NOTSUPPORTED"                : "{APP_NAME} comando di installazione strumento linea non è supportato su questo SO.",
-
-    "LAUNCH_SCRIPT_CREATE_SUCCESS"              : "Strumento a riga di comando installato correttamente! Ora si può facilmente lanciare {APP_NAME} dalla riga di comando utilizzando <code>{APP_NAME} myFile.txt</code> o <code>{APP_NAME} myFolder</code>. <br/><br/>Per favore riferisci a <a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments'>linea du comando</a> wiki per maggiori informazioni.",
-    "CREATING_LAUNCH_SCRIPT_TITLE"              : "Aggiungi la linea di comando di Brackets",
+    "ERROR_CLTOOLS_MKDIRFAILED"                 : "Impossibile creare la cartella <code>/usr/local/bin</code>.",
+    "ERROR_CLTOOLS_LNFAILED"                    : "Impossibile creare il collegamento simbolico <code>/usr/local/bin/brackets</code>.",
+    "ERROR_CLTOOLS_SERVFAILED"                  : "Errore interno.",
+    "ERROR_CLTOOLS_NOTSUPPORTED"                : "Collegamento Riga di comando non è supportato su questo SO.",
```

Comment:
I think OS is better. (I corrected some other strings which were translated *too literally*)